test/benchmarks/material/card/card.perf-spec.ts

```diff
@@ -0,0 +1,28 @@
+/**
+ * @license
+ * Copyright Google Inc. All Rights Reserved.
+ *
+ * Use of this source code is governed by an MIT-style license that can be
+ * found in the LICENSE file at https://angular.io/license
+ */
+
+import {$, browser} from 'protractor';
+import {runBenchmark} from '@angular/dev-infra-private/benchmark/driver-utilities';
+
+describe('card performance benchmarks', () => {
+  beforeAll(() => {
+    browser.rootEl = '#root';
+  });
+
+  it('renders a simple card', async() => {
+    await runBenchmark({
+      id: 'card-render',
+      url: '',
+      ignoreBrowserSynchronization: true,
+      params: [],
+      setup: async () => await $('#show').click(),
```

Comment:
Same as in your others PRs. Why would we want to click `#show` initially when we call `#hide` first on every sample iteration anyway? Seems unnecessary or do I miss something?